release_notes/ocp_3_11_release_notes.adoc

```diff
@@ -325,7 +328,12 @@ The new registry, `registry.redhat.io`, requires authentication for access to
 images and hosted content on {product-title}. Following the move to the new
 registry, the existing registry will be available for a period of time.
 
-See Accessing and Configuring the Red Hat Registry for more information.
+See xref:../../install_config/registry/index.adoc#auth-enabled-registry[Authentication Enabled Red Hat Registry]
+for more information. 
+
```

Comment:
This link won't work until https://github.com/openshift/openshift-docs/pull/11506 is merged.